app/src/main/java/com/bignerdranch/android/geoquiz/MainActivity.kt

```diff
@@ -56,13 +62,35 @@ class MainActivity : AppCompatActivity() {
     }
 
     private fun checkAnswer(userAnswer: Boolean) {
-        val correctAnswer = questionBank[currentIndex].answer
-        val messageResId = if (userAnswer == correctAnswer) {
-            R.string.correct_toast
-        } else {
-            R.string.incorrect_toast
-        }
-        Toast.makeText(this, messageResId, Toast.LENGTH_SHORT)
-            .show()
+//        val correctAnswer = questionBank[currentIndex].answer
+//        val messageResId = if (userAnswer == correctAnswer) {
+//            R.string.correct_toast
+//        } else {
+//            R.string.incorrect_toast
+//        }
+//        Toast.makeText(this, messageResId, Toast.LENGTH_SHORT)
+//            .show()
+//    }
+//
+        val correctAnswer=questionBank[currentIndex].answer
+        val messageAnswer=if (userAnswer == correctAnswer)R.string.correct_toast else R.string.incorrect_toast
+
+        val toast1=Toast.makeText(this,getString(messageAnswer),Toast.LENGTH_SHORT)
+        val _toast1:TextView = toast1.view!!.findViewById(android.R.id.message)
+        toast1.setGravity(Gravity.CENTER,5,5)
```

Comment:
Nice work but the request in part 1 is to make a toast at the top, not at the center.